src/test/regress/expected/multi_extension.out

```diff
@@ -902,6 +902,59 @@ SELECT * FROM multi_extension.print_extension_changes();
                                                                       | function worker_fix_partition_shard_index_names(regclass,text,text) void
 (6 rows)
 
+-- Test downgrade to 11.0-1 from 11.0-2
+ALTER EXTENSION citus UPDATE TO '11.0-2';
+ALTER EXTENSION citus UPDATE TO '11.0-1';
+-- Should be empty result since upgrade+downgrade should be a no-op
+SELECT * FROM multi_extension.print_extension_changes();
+ previous_object | current_object
+---------------------------------------------------------------------
+(0 rows)
+
+-- Make sure that we delete pg_depend entries added in
+-- columnar--11.0-1--11.0-2.sql when downgrading to 11.0-1.
+SELECT COUNT(*)
+FROM pg_depend
+WHERE classid = 'pg_am'::regclass::oid AND
+      objid = (select oid from pg_am where amname = 'columnar') AND
+      objsubid = 0 AND
+      refclassid = 'pg_class'::regclass::oid AND
+      refobjsubid = 0 AND
+      deptype = 'n';
+ count
+---------------------------------------------------------------------
+     0
+(1 row)
+
+-- Snapshot of state at 11.0-2
+ALTER EXTENSION citus UPDATE TO '11.0-2';
+SELECT * FROM multi_extension.print_extension_changes();
+ previous_object | current_object
+---------------------------------------------------------------------
+(0 rows)
+
+-- Make sure that we defined dependencies from all rel objects (tables,
+-- indexes, sequences ..) to columnar table access method ...
+SELECT pg_class.oid INTO columnar_schema_members
+FROM pg_class, pg_namespace
+WHERE pg_namespace.oid=pg_class.relnamespace AND
+      pg_namespace.nspname='columnar';
+SELECT refobjid INTO columnar_schema_members_pg_depend
+FROM pg_depend
+WHERE classid = 'pg_am'::regclass::oid AND
+      objid = (select oid from pg_am where amname = 'columnar') AND
+      objsubid = 0 AND
+      refclassid = 'pg_class'::regclass::oid AND
+      refobjsubid = 0 AND
+      deptype = 'n';
+-- ... so, this should be empty.
+(TABLE columnar_schema_members EXCEPT TABLE columnar_schema_members_pg_depend)
```

Comment:
would that make sense to also show that `columnar_schema_members` and `columnar_schema_members_pg_depend` have non-zero elements?

Reply:
yeah makes sense, I can run two count(*) queries for those